android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/transition/Transition.kt

```diff
@@ -0,0 +1,60 @@
+package com.badoo.ribs.core.routing.transition
+
+import android.animation.ValueAnimator
+
+interface Transition {
+
+    fun start()
+
+    fun end()
+
+    fun pause()
+
+    fun reverse()
+
+    companion object {
+        fun multiple(vararg transitions: Collection<Transition?>) = object : Transition {
+            override fun start() {
+                transitions.flatMap {
```

Comment:
why flatMap if you don't return result?
Maybe `transition.forEach {  it.forEach { it.start() } }`

Reply:
maybe better to use 
```
transitions.flatten().forEach { it.start() }
```

If flatten is supported for vararg arguments.

Reply:
Check is it possible to use flatten here.
```transitions.flatten().forEach { it.start() }```

Reply:
flatten not available, fixed with foreach